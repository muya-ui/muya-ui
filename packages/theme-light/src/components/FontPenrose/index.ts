export { default } from './FontPenrose';

export { default as FontPenrose } from './FontPenrose';
export * from './FontPenrose';
