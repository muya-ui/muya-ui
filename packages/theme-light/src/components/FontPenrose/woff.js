/* eslint-disable */
const fs = require('fs');
const path = require('path');

const componentPath = path.join(__dirname, 'FontPenrose.tsx');
const woffBuffer = fs.readFileSync(path.join(__dirname, 'Penrose.woff'));
const woffBase64 = woffBuffer.toString('base64');

const penroseComponentStr = fs.readFileSync(componentPath, 'utf8');

const buildStr = penroseComponentStr.replace(
  /penroseWoffBase64Str[^;]*;/,
  `penroseWoffBase64Str = '${woffBase64}';`,
);

fs.writeFileSync(componentPath, buildStr, 'utf8');

console.log('penrose replace done!');
