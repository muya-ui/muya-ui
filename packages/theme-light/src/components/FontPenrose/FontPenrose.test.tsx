import React from 'react';
import renderer from 'react-test-renderer';

import FontPenrose from './FontPenrose';

test('测试 FontPenrose', () => {
  const tree = renderer.create(<FontPenrose />).toJSON();
  expect(tree).toMatchSnapshot();
});
