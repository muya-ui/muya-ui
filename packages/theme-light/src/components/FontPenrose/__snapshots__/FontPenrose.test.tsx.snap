// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`测试 FontPenrose 1`] = `null`;
